end-to-end/e2e.spec.ts

```diff
@@ -298,6 +298,38 @@ describe('e2e', function () {
       expect((await ethCompoundPCVDeposit.balance()).sub(balanceBefore)).to.be.bignumber.lessThan(amount);
     })
   })
+
+  describe('Aave', async () => {  
+    it('should be able to deposit and withdraw ETH',  async function () {
+      const aaveEthPCVDeposit = contracts.aaveEthPCVDeposit;
+      const amount = tenPow18.mul(toBN(200));
+      await web3.eth.sendTransaction({from: deployAddress, to: aaveEthPCVDeposit.address, value: amount });
+
+      const balanceBefore = await aaveEthPCVDeposit.balance();
+
+      await aaveEthPCVDeposit.deposit();
+      expectApprox((await aaveEthPCVDeposit.balance()).sub(balanceBefore), amount, '100');
+
+      await aaveEthPCVDeposit.withdraw(deployAddress, amount);
+
+      expect((await aaveEthPCVDeposit.balance()).sub(balanceBefore)).to.be.bignumber.lessThan(amount);
+    })
+
+    it('should be able to earn and claim stAAVE', async () => {
+      const aaveEthPCVDeposit = contracts.aaveEthPCVDeposit;
+      const amount = tenPow18.mul(toBN(200));
+      await web3.eth.sendTransaction({from: deployAddress, to: aaveEthPCVDeposit.address, value: amount });
+
+      const aaveBalanceBefore = await contracts.stAAVE.balanceOf(aaveEthPCVDeposit.address);
+      await aaveEthPCVDeposit.deposit();
+
+      await aaveEthPCVDeposit.claimRewards();
+      const aaveBalanceAfter = await contracts.stAAVE.balanceOf(aaveEthPCVDeposit.address);
+
+      expect(aaveBalanceAfter.sub(aaveBalanceBefore)).to.be.bignumber.greaterThan(toBN(0));
```

Comment:
How come the rewards balance is not 0 if the deposit just happened ?

I guess you could add a small time fast-forward in your test, to check that you earn some interest?

Reply:
Time did fast forward at least 1 second/block between the deposit and the claim